internal/pkg/cli/pipeline_delete.go

```diff
@@ -176,21 +179,34 @@ func (o *deletePipelineOpts) askAppName() error {
 }
 
 func (o *deletePipelineOpts) getSecret() error {
-	path, err := o.ws.PipelineManifestLegacyPath()
-	if err != nil {
-		return fmt.Errorf("get path to pipeline manifest: %w", err)
+	// Look for default secret name for GHv1 access token based on default pipeline name.
+	o.ghAccessTokenSecretName = o.pipelineSecretName()
+	output, err := o.secretsmanager.DescribeSecret(o.ghAccessTokenSecretName)
+	if err == nil {
+		// Check for Copilot-assigned tag for added assurance. If tags not found, fall through.
+		for _, tag := range output.Tags {
+			if aws.StringValue(tag.Key) == "copilot-application" && aws.StringValue(tag.Value) == output.CreatedDate.UTC().Format(time.UnixDate) {
+				log.Infof("Found secret '%s'.\n", o.ghAccessTokenSecretName)
+				return nil
+			}
+		}
 	}
-	manifest, err := o.ws.ReadPipelineManifest(path)
-	if err != nil {
-		return fmt.Errorf("read pipeline manifest: %w", err)
-	}
-
-	if secret, ok := (manifest.Source.Properties["access_token_secret"]).(string); ok {
-		o.ghAccessTokenSecretName = secret
+	var notFoundErr *secretsmanager.ErrSecretNotFound
+	if err != nil && !errors.As(err, &notFoundErr) {
+		return fmt.Errorf("describe secret %s: %w", o.ghAccessTokenSecretName, err)
 	}
+	// To get here, either the secret was found but tags didn't match, or Secrets Manager returned a ResourceNotFoundException.
+	log.Infof("Found no Copilot-generated Secrets Manager secrets to delete for pipeline '%s'.\n", o.name)
```

Comment:
Samesies here!

Reply:
Yes-- rather than adding more explanation to this one, I'll just delete it, as most current users will fall into this category.